python/vmaf/core/vmafexec_feature_extractor.py

```diff
@@ -161,6 +247,38 @@ class IntegerVifFeatureExtractor(VmafexecFeatureExtractorMixin, FeatureExtractor
         'vif_den_scale2': 'integer_vif_den_scale2',
         'vif_num_scale3': 'integer_vif_num_scale3',
         'vif_den_scale3': 'integer_vif_den_scale3',
+
+        'vif_scale0_egl_1.00': 'integer_vif_scale0_egl_1.00',
```

Comment:
This is a bit ugly. You can keep it this way first and I can simply the code later.

Reply:
Yeah, I figured there is a cleaner way to do this.